### ISSUE TITLE
locations (and other associations) do not reload on entity page if entity not changed

### ISSUE THREAD

**Issue by jessib** [public-accountability/littlesis-rails#1756]
I'm not sure if this is true for other associations, but if I change only the location(s) for an entity, it doesn't display the updated locations until I change something on the entity (or people table?) itself

**Comment by aepyornis** [comment]
Yup, to invalidate the entity page change we have to update the "updated_at" timestamp. We probably want to call `entity.touch` somewhere. Alias is using a callback: https://github.com/public-accountability/littlesis-rails/blob/main/app/models/alias.rb#L17
